addons/account/migrations/9.0.1.1/pre-migration.py

```diff
@@ -249,6 +249,13 @@ def map_account_tax_template_type(cr):
         table='account_tax_template', write='sql')
 
 
+def map_payment_term_line_value(cr):
+    """ Someone fixed a Flemishism """
+    openupgrade.logged_query(
+        """UPDATE account_payment_term_line
```

Comment:
Fixed now. I tend not to commit these small fixes untested because I rely on Travis, but I don't want it to be more work for you going into the travis build and look around. So if you like, you can just ignore these red PRs of mine.

Reply:
There's not too much problem, but as you prefer.